types/pi-pir-sensor/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for pi-pir-sensor 0.0
+// Project: https://github.com/opedromiranda/pi-pir-sensor
+// Definitions by: Mario Murrent <https://github.com/meecodebymariomurrent>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function Sensor(configuration: Sensor.SensorConfiguration): void;
+
+declare namespace Sensor {
+    class Sensor {
+        lastMovement: Date;
+
+        constructor(sensorConfiguration: SensorConfiguration);
+
+        start(): void;
```

Comment:
this one IMO accepts callback `(err) => void`:
https://github.com/opedromiranda/pi-pir-sensor/blob/master/lib/sensor.js#L30